docs/projects/ad7606x_fmcz/index.rst

```diff
@@ -0,0 +1,450 @@
+
+The INTF configuration parameter defines the interface type (Serial or
+Parallel). By default it is set to 0. Depending on the required interface mode,
+some hardware modifications need to be done on the board and/or make command:
+
+In case of the **PARALLEL** interface:
+
+.. code-block::
+
+   make INTF=0
+
+In case of the **SERIAL** interface:
+
+.. code-block::
+
+   make INTF=1
+
+.. note::
+
+   This switch is a *hardware* switch. Please rebuild the  design if the
+   variable has been changed.
+
+   -   JP5 - Position A - Serial interface
+   -   JP5 - Position B - Parallel interface
+
+The DEV_CONFIG configuration parameter defines the device which will be used:
+-  Options: 0 - AD7606B, 1 - AD7606C-16, 2 - AD7606C-18. 
+By default it is set to 0.
+
+The NUM_OF_SDI configuration parameter defines the number of SDI lines used:
+-  Options: 1, 2, 4, 8. 
+By default is set to 8.
+
+The SIMPLE_STATUS_CRC configuration parameter defines the ADC Read Mode used:
+-  Options: SIMPLE(0), STATUS(1), CRC(2), CRC_STATUS(3). 
+By default is set to 0.
+
+The EXT_CLK configuration parameter defines the external clock option for
+the ADC clock:
+-  Options: 0 - No, 1 - Yes. 
+By default is set to 0.
+
+Jumper setup
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+================== ================ ====================================
+Jumper/Solder link Default Position Description
+================== ================ ====================================
+JP1                A                The STBY pin is tied to VDRIVE
+JP2                A                12V supply from the carrier
+JP3                A                3.3V supply from the ADP7118
+JP4                A                ±10V range is selected
+JP5                A                Serial interface
+JP6                B                The internal reference is disabled, and the external 
+                                    reference is selected. P5 must be inserted if using 
+                                    the on-board U1 device
+JP7                B                Connects the V1- line to the J5 SMB connector
+JP8,JP10           A                Bypasses the amplifier mezzanine card
+JP9,JP11           A                Bypasses the amplifier mezzanine card
+JP12               B                Connects the V2- line to the J6 SMB connector
+JP13               B                Connects the V3- line to the J7 SMB connector
+JP14               B                Connects the V4- line to the J8 SMB connector
+P13                Inserted         Connects the V5- line to ground
+P14                Inserted         Connects the V8- line to ground
+P15                Inserted         Connects the V6- line to ground
+P16                Inserted         Connects the V7- line to ground
+S1                 On               Controls the OS0, OS1, and OS2 pins
+================== ================ ====================================
+
+IP list
```

Comment:
The `IP list` section should be removed, as these IPs are already present in the `HDL related` section